tests/perf/README.md

```diff
@@ -52,22 +60,27 @@ Vegeta request files are written to `/tmp/turbo_geth_stress_test`:
 From Silkrpc project directory check the performance test runner usage:
 ```
 $ tests/perf/run_perf_tests.py
-Usage: ./run_perf_tests.py vegetaPatternTarFile [daemonOnCore] [turboGethAddress] [turboGethHomeDir] [testRepetitions] [testSequence]
+Usage: ./run_perf_tests.py -h -p vegetaPatternTarFile -c daemonOnCore  -t turboErigonAddress -g turboErigonHomeDir -s silkrpcBuildDir -r testRepetitions - t testSequence
 
 Launch an automated performance test sequence on Silkrpc and RPCDaemon using Vegeta
 
-vegetaPatternTarFile     path to the request file for Vegeta attack
-daemonOnCore             logical cpu list in taskset format (e.g. - or 0-1 or 0-2 or 0,2...)              [default: -]
-turboGethAddress         address of TG Core component as <address>:<port> (e.g. localhost:9090)           [default: localhost:9090]
-turboGethHomeDir         path to TG home folder (e.g. ../../../turbo-geth/)                               [default: ../../../turbo-geth/]
-testRepetitions          number of repetitions for each element in test sequence (e.g. 10)                [default: 10]
-testSequence             list of query-per-sec and duration tests as <qps1>:<t1>,... (e.g. 200:30,400:10) [default: 50:30,200:30,200:60,400:30]
+-h                      print this help
+-d rpcDaemonAddress     address of daemon eg (10.1.1.20)                                                       [default: localhost]
+-p vegetaPatternTarFile path to the request file for Vegeta attack                                             [default: ./vegeta/erigon_stress_test_001.tar]
+-c daemonVegetaOnCore   cpu list in taskset format for daemon & vegeta (e.g. 0-1:2-3 or 0-2:3-4 or 0,2:3,4...) [default: -:-]
+-a erigonAddress        address of ERIGON Core component as <address>:<port> (e.g. localhost:9090)             [default: localhost:9090]
+-g erigonHomeDir        path to ERIGON home folder (e.g. ../../../erigon/)                                     [default: ../../../erigon/build/]
```

Comment:
The parameter name is `erigonHomeDir` that sounds like the Erigon project home folder as indicated in the example `(e.g. ../../../erigon/)` but then the default value is `[default: ../../../erigon/build/]` which seems like the build output directory. So either the parameter name is misleading or the default value is wrong here

Reply:
fixed